src/debug/seer-integration.js

```diff
@@ -0,0 +1,91 @@
+import seer from 'seer';
+import {window} from '../lib/utils/globals';
+
+const overrides = new Map();
+
+/**
+ * Create an override on the specify layer, indexed by a valuePath array.
+ * Do nothing in case Seer as not been initialized to prevent any preformance drawback.
+ */
+export const setOverride = (id, valuePath, value) => {
+  if (!window.__SEER_INITIALIZED__) {
+    return;
+  }
+
+  if (!overrides.has(id)) {
+    overrides.set(id, new Map());
+  }
+
+  const props = overrides.get(id);
+  props.set(valuePath, value);
+
+};
+
+/**
+ * Recursively set a nested property of an object given a properties array and a value
+ */
+const recursiveSet = (obj, path, value) => {
+  if (path.length > 1) {
+    recursiveSet(obj[path[0]], path.slice(1), value);
+  } else {
+    obj[path[0]] = value;
+  }
+};
+
+/**
+ * Get the props overrides of a specific layer if Seer as been initialized
+ * Invalidates the data to be sure new ones are always picked up.
+ */
+export const getOverrides = props => {
```

Comment:
They were, but the `recursiveSet` util that is unexported needed to be declared first, I can move it at the very top though